test/common/verify_alert_links_to_SOPs.go

```diff
@@ -0,0 +1,147 @@
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	prometheusv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+)
+
+var githubToken = os.Getenv("TOKEN_GITHUB")
```

Comment:
Sure no problem, but can you please explain further, why exactly it would be better? I got inspiration from another test cases, where credentials were declared as global variables

Reply:
Fair question. In general the variable should only be visible where it is needed. If you need it in just one function then it should be declared in that function and not globally. If you need it in many functions then it is better to have it declared/defined globally so that you don't have to pass it into that many functions via input parameter. And then there is a grey zone in between.

In this particular case you need it in two functions, it is not that many so you can just declare/define it in one and pass via input parameter to the other. However, it is fairly ok as is too, it mostly depends on personal taste.